File: Front/ProEventos-App/src/app/models/Evento.ts
import { RedeSocial } from './RedeSocial';
import { Lote } from './Lote';
import { PalestranteEvento } from './PalestranteEvento';


export interface Evento {
  id: number,
  local: string,
  dataEvento: Date,
  tema: string,
  qtdPessoas: number,
  imagemURL: string,
  telefone: string,
  email: string,
  lotes: Lote[],
  redesSociais: RedeSocial[],
  palestrantesEventos: PalestranteEvento[]
}

File: Back/src/ProEventos.Domain/Models/Evento.cs
﻿using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;
using System.Linq;
using System.Threading.Tasks;

namespace ProEventos.Domain.Models
{
    public class Evento
    {
        public int Id { get; set; }
        public string Local { get; set; }
        public DateTime? DataEvento { get; set; }

        [Required]
        public string Tema { get; set; }
        public int QtdPessoas { get; set; }
        public string ImagemURL { get; set; }
        public string Telefone { get; set; }
        public string Email { get; set; }
        public IEnumerable<Lote> Lotes { get; set; }
        public IEnumerable<RedeSocial> RedesSociais { get; set; }
        public IEnumerable<PalestranteEvento> PalestrantesEventos { get; set; }

    }
}
